Doc/library/netrc.rst

```diff
@@ -20,8 +20,9 @@ the Unix :program:`ftp` program and other FTP clients.
 
    A :class:`~netrc.netrc` instance or subclass instance encapsulates data from  a netrc
    file.  The initialization argument, if present, specifies the file to parse.  If
-   no argument is given, the file :file:`.netrc` in the user's home directory will
-   be read.  Parse errors will raise :exc:`NetrcParseError` with diagnostic
+   no argument is given, the file :file:`.netrc` in the user's home directory --
+   as returned by :func:`os.path.expanduser` -- will be read.
```

Comment:
Maybe "as determined"? `expanduser()` itself doesn't return the user's home directory.

Or "as returned by `os.path.expanduser('~')` ", but this exposes more implementation details and is harder to format as a reference.